src/commands/app-configs/undeploy.ts

```diff
@@ -67,19 +66,15 @@ found in the JSON output of the app-configs:get command.`
 
     for (const undeployment of undeployments) {
       const {
-        applicationConfigDeploymentId: deploymentId,
-        configId,
-        environmentGeographyId,
+        message,
       } = undeployment
 
-      this.log(`Application configuration ID ${chalk.cyan(configId)} undeployed from ` +
-        `environmentGeography ID ${chalk.cyan(environmentGeographyId)} ` +
-        `with deployment ID ${chalk.cyan(deploymentId)}.`)
+      this.log(message)
     }
   }
 
   setRequestActionMessage(_options: any) {
     debug('setRequestActionMessage()')
-    this.requestActionMessage = 'Undeploying application configuration'
+    this.requestActionMessage = 'Undeploying application configuration ID from environmentGeography ID with deployment ID '
```

Comment:
The requestActionMessage needs to be tailored to the flags provided by the user.
If the user provides the 'env-geo' flag then the message should say:
"Undeploying application configuration <configID> from environmentGeography ID <env-geo-id>"

If the user does not provide the 'env-geo' flag then the message should say:
"Undeploying application configuration <configID>"